scripts/install-native.mjs

```diff
@@ -0,0 +1,54 @@
+import { execFile } from 'child_process'
+import fs from 'fs/promises'
+import fse from 'fs-extra'
+import os from 'os'
+import path from 'path'
+import util from 'util'
+
+const exec = util.promisify(execFile)
+
+;(async function () {
+  try {
+    let tmpdir = os.tmpdir()
```

Comment:
```suggestion
    let tmpdir = path.join(os.tmpdir(), `next-swc-${Date.now()}`)
```
Should we use a nested folder in temp so we don't install directly to the temp directory?